spec/parseFilterQuery.spec.ts

```diff
@@ -67,7 +67,12 @@ describe("parseFilterQuery", () => {
     const clientId = "619af485cd56547936847584";
     let bookmarked = true;
     const result1 = parseBalanceQuery({ clientId, bookmarked }, { name: "MyBook" });
-    expect(result1).to.deep.equal({ book: "MyBook", meta: { clientId, bookmarked } });
+    expect(result1).to.deep.equal({
+      book: "MyBook",
+      "meta.bookmarked": bookmarked,
+      "meta.clientId": clientId,
+      meta: { clientId, bookmarked },
```

Comment:
Can you please explain? I would actually expect to have either dot notation or "normal" search query as filter document.